src/functionaltests/java/com/ericsson/ei/subscriptions/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -0,0 +1,229 @@
+package com.ericsson.ei.subscriptions.query;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Ignore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get; 
+
+import com.ericsson.ei.mongodbhandler.MongoDBHandler;
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+
+
+@Ignore
+@AutoConfigureMockMvc
+public class QueryAggregatedObjectsTestSteps extends FunctionalTestBase {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(QueryAggregatedObjectsTestSteps.class);
+
+    
+    private static final String aggregatedObjJsonPath = "src/test/resources/AggregatedObject.json";
+    private static final String missedNotificationJsonPath = "src/test/resources/MissedNotification.json";
+
+    
+    @Autowired
+    private MockMvc mockMvc;
+    private MvcResult mvcResult;
+    
+    @LocalServerPort
+    private int port;
+    
+    @Autowired
+    private MongoDBHandler mongoDBHandler;
+    
+    @Value("${spring.data.mongodb.database}")
+    private String eiDatabseName;
+    
+    @Value("${aggregated.collection.name}")
+    private String aggrCollectionName;
+    
+    @Value("${missedNotificationCollectionName}")
+    private String missedNotificationCollectionName;
+    
+    @Value("${missedNotificationDataBaseName}")
+    private String missedNotificationDatabaseName;
+
+    private String aggrObj;
+    private String missedNotificationObj;
+    
+    private ObjectMapper objMapper;
```

Comment:
you use ObjectMapper many times in code below so i think you can create ObjectMapper instance here